archetypes/database-mp/src/main/resources/src/main/resources/native-image/reflect-config.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Intentional?

Reply:
We don't really need at this time. I will remove it in a follow up PR.